tools/pylint-extensions/azure-pylint-guidelines-checker/tests/test_pylint_custom_plugins.py

```diff
@@ -4422,11 +4433,10 @@ def test_docstring_varag_no_type(self):
         # Error on documenting keyword only args as param after *args in docstring
         node = astroid.extract_node(
             """
-            def function_foo(*x, y, z):
```

Comment:
flaky test fix :)

Reply:
do a set() ^ set() causes variables to flip flop -- doesnt effect the actual rule itself just the test catch